src/fides/api/models/privacy_preference_v2.py

```diff
@@ -215,12 +215,6 @@ def privacy_experience_config_history_id(cls) -> Column:
             index=True,
         )
 
-    # The specific experience under which the user was presented the relevant notice
-    # Minimal information stored here, mostly just region
-    privacy_experience_id = Column(
-        String
-    )  # TODO will be removed.  Preemptively removing the FK to Privacy Experience
-
```

Comment:
I'm going to be looking into these types of fields here https://ethyca.atlassian.net/browse/PROD-1776